package/com.unity.formats.usd/Documentation~/USD.md

```diff
@@ -114,3 +114,8 @@ We do not currently support the following:
     * Blend Shapes
 * General:
     * Custom prims
+
+
+## Analytics
+
+We collect high level analytics of the features used in the package, to help us understand adoption and prioritize future development work. You can see exactly what data points are collected in `Common\UsdEditorAnalytics.cs`.
```

Comment:
Comment: Maybe something for future, but maybe there is a contact email for analytics at Unity so as/if corporate entities have concerns over our collections they can make queries to address the concern that could arise from analytics being present. i.e. Opt-in or out GDPR sort of stuff

Reply:
These are just standard analytics the same as any other in the Unity Editor, as described in the official documentation: https://docs.unity3d.com/Manual/EditorAnalytics.html